src/components/pages/LockScreen.tsx

```diff
@@ -73,13 +93,27 @@ export const LockScreen = (): JSX.Element => {
           />
 
           <MenuItemSwitch
-            primary={t('Nav.primary_lock_switch')}
+            checked={pinCodeEnabled}
+            icon={Password}
+            onClick={(): void => void onPinCodeLock()}
```

Comment:
Same warning as in previous PR, we should find a way to just put pointers on this prop. We don't want to declare an anonymous method just to satisfy typescript linter.

Reply:
I'm still trying to find a way to convert onClick handler to an async call but TypeScript is smarter than me and always finds that I hid a promise somewhere in the callback.

Reply:
Maybe we should remove this specific rule then? I'm happy with all the TS checks, but I don't like the fact that we have to modify the app's behaviour just to pass them.